cmd/kured/main.go

```diff
@@ -90,6 +93,11 @@ func main() {
 	rootCmd.PersistentFlags().StringVar(&slackChannel, "slack-channel", "",
 		"slack channel for reboot notfications")
 
+	rootCmd.PersistentFlags().StringVar(&notifyURL, "notify-url", "",
+		"a formatted url to notify accross chat types")
+	rootCmd.PersistentFlags().StringVar(&notifyMessage, "notify-message", "",
```

Comment:
I don't think we should expose a custom notify message yet. I think it is relevant for another (follow-up) PR.

Reply:
Removed. Thanks for the feedback.